src/main/kotlin/app/revanced/patches/youtube/layout/buttons/action/patch/HideButtonsPatch.kt

```diff
@@ -51,12 +51,21 @@ class HideButtonsPatch : ResourcePatch {
                         StringResource("revanced_hide_playlist_button_summary_on", "Playlist button is hidden"),
                         StringResource("revanced_hide_playlist_button_summary_off", "Playlist button is shown")
                     ),
+                    SwitchPreference(
+                        "revanced_hide_clip_button",
+                        StringResource("revanced_hide_clip_button_title", "Hide clip button"),
+                        false,
+                        StringResource("revanced_hide_clip_button_summary_on", "Clip button is hidden"),
+                        StringResource("revanced_hide_clip_button_summary_off", "Clip button is shown"),
+                        StringResource("revanced_hide_clip_button_user_dialog_message",
+                            "Hide clip button only works for some users and some devices.  If the clip button is still visible, then it can only be hidden by enabling \\'Hide all other action buttons\\'")
```

Comment:
```suggestion
                            "Hiding the clip button might not work reliably. In the case it does not work, it can only be hidden by enabling \\'Hide all other action buttons\\'''")
```